### PR TITLE
ROX-19857: long running collector should have listening endpoints load

### PR DESCRIPTION
**Pull request by JoukoVirtanen**
## Description

Kube-burner now creates listening endpoints load via berserker. A new deployment is created in which berserker reads a workload file that specifies the endpoints load. The workload used for the endpoints is specified in a confimap.

## Checklist
- [x] Investigated and inspected CI test results
~~- [ ] Unit test and regression tests added~~
~~- [ ] Evaluated and added CHANGELOG entry if required~~
~~- [ ] Determined and documented upgrade steps~~
~~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

If any of these don't apply, please comment below.

This just changes some config files for kube-burner, so no testing or user facing documentation is required. 

## Testing Performed

Go to https://github.com/stackrox/test-gh-actions/actions/workflows/create-clusters.yml

Click on "Run workflow".
Change branch to jv-ROX-19857-long-running-collector-should-have-listening-endpoints-load

Select "Create a long-running cluster on RC1" and "Create a GKE demo cluster"
Click run workflow.

The names of created clusters are "real-load-4-1-1" and "fake-load-4-1-1".
Download the artifacts for the clusters and use their kubeconfigs.

Connect to the central-db in the cluster running the fake workload. Check the number of listening endpoints.

central_active=# select count(*) from listening_endpoints ;
-[ RECORD 1 ]
count | 20987

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: scripts/release-tools/kube-burner-configs/cluster-density/config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ global:
 jobs:
   - name: cluster-density
     namespace: cluster-density
-    jobIterations: 10
+    jobIterations: 5
```
Comment (JoukoVirtanen) [inline scripts/release-tools/kube-burner-configs/cluster-density/config.yml:14]: There are now two types of deployments so cut the number of jobIterations in half to keep the same number of deployments.

File: scripts/release-tools/kube-burner-configs/cluster-density/endpoint-load-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      app: endpoint-load-{{.Replica}}
+  template:
+    metadata:
+      labels:
+        app: endpoint-load-{{.Replica}}
+    spec:
+      imagePullSecrets:
+      - name: {{.JobName}}-{{.Replica}}
+      containers:
+      - image: quay.io/rhacs-eng/qa:berserker-1.0-38-ge3b2927865
```
Comment (JoukoVirtanen) [inline scripts/release-tools/kube-burner-configs/cluster-density/endpoint-load-deployment.yml:19]: This berserker image is not a main image.

Comment (tommartensen) [reply scripts/release-tools/kube-burner-configs/cluster-density/endpoint-load-deployment.yml:19]: What do you mean by that?

Comment (JoukoVirtanen) [reply scripts/release-tools/kube-burner-configs/cluster-density/endpoint-load-deployment.yml:19]: This image is from a PR not the main branch. That PR is now merged so I will fix this soon and use an image built from the main branch.

Comment (tommartensen) [reply scripts/release-tools/kube-burner-configs/cluster-density/endpoint-load-deployment.yml:19]: Let us know when that is fixed, so we can review this PR.

Comment (JoukoVirtanen) [reply scripts/release-tools/kube-burner-configs/cluster-density/endpoint-load-deployment.yml]: I have updated the berserker image tag.

**Review by tommartensen — COMMENTED** [review]
From my side (release improvements), nothing speaks against merging this PR. 
I'll let the other two reviewers comment on the specifics.

**Review by erthalion — APPROVED** [review]
Looks good to me as well. We could probably even parametrize the config to make everything a bit more flexible, but it could be done later.

**Pull request merged**